reference/3.0/Microsoft.PowerShell.Core/About/about_CommonParameters.md

```diff
@@ -14,448 +14,344 @@ Describes the parameters that can be used with any cmdlet.
 
-Valid values:
+The **Verbose** parameter overrides the value of the `$VerbosePreference`
+variable for the current command. Because the default value of the
+`$VerbosePreference` variable is **SilentlyContinue**, verbose messages aren't
+displayed by default.
 
-- $true (-Verbose:$true) has the same effect as -Verbose.
+`-Verbose:$true` has the same effect as `-Verbose`
 
-- $false (-Verbose:$false) suppresses the display of verbose messages. Use
-  this parameter when the value of $VerbosePreference is not SilentlyContinue
-  (the default).
+`-Verbose:$false` suppresses the display of verbose messages. Use this parameter
+when the value of `$VerbosePreference` isn't **SilentlyContinue** (the
+default).
 
--WarningAction[:{Continue | Inquire | SilentlyContinue | Stop}]
+#### WarningAction
 
-Alias: wa
+The alias for **WarningAction** is **wa**.
 
-Determines how the cmdlet responds to a warning from the command. "Continue"
+Determines how the cmdlet responds to a warning from the command. **Continue**
 is the default value. This parameter works only when the command generates a
 warning message. For example, this parameter works when a command contains the
-Write-Warning cmdlet.
+`Write-Warning` cmdlet.
 
-The WarningAction parameter overrides the value of the $WarningPreference
-variable for the current command. Because the default value of the
-$WarningPreference variable is Continue, warnings are displayed and execution
-continues unless you use the WarningAction parameter.
-
-Valid Values:
+The **WarningAction** parameter overrides the value of the
+`$WarningPreference` variable for the current command. Because the default
+value of the `$WarningPreference` variable is **Continue**, warnings are
+displayed and execution continues unless you use the **WarningAction**
+parameter.
 
-- Continue. Displays the warning message and continues executing the command.
-  "Continue" is the default value.
+`-WarningAction:Continue` displays the warning messages and continues executing
+the command. `Continue` is the default.
 
-- Inquire. Displays the warning message and prompts you for confirmation
-  before continuing execution. This value is rarely used.
+`-WarningAction:Inquire` displays the warning message and prompts you for
+confirmation before continuing execution. This value is rarely used.
 
-- SilentlyContinue. Suppresses the warning message and continues executing the
-  command.
+`-WarningAction:SilentlyContinue` suppresses the warning message and continues
+executing the command.
 
-- Stop. Displays the warning message and stops executing the command.
+`-WarningAction:Stop` displays the warning message and stops executing the
+command.
 
-NOTE: The WarningAction parameter does not override the value of the
-$WarningAction preference variable when the parameter is used in a command to
-run a script or function.
+> [!NOTE]
+> The `-WarningAction` parameter does not override the value of the
```

Comment:
I think we need to change the wording of this. It is not accurate. This may apply to the other Action parameters as well.